lib/api/field-locale.js

```diff
@@ -1,18 +1,20 @@
-import Signal from './signal'
+import { MemoizedSignal } from './signal'
 
 export default class FieldLocale {
 
   constructor (channel, {id, locale, value}) {
     this.id = id
     this.locale = locale
     this._value = value
-    this._valueSignal = new Signal()
+    this._valueSignal = new MemoizedSignal(value)
     this._channel = channel
 
     channel.addHandler('valueChanged', (id, locale, value) => {
       if (id === this.id && (!locale || locale === this.locale)) {
-        this._value = value
-        this._valueSignal.dispatch(value)
+        if (this._value !== value) {
```

Comment:
Why a nested `if`. Should all be part of one if. But please separate the conditions into variables.